src/pages/TakeTurnPage.vue

```diff
@@ -93,10 +94,21 @@ const onBodegasGetted = (bodegas: Bodega[]) => {
   bodegaSelect.value.options.length = 0;
   bodegaSelect.value.selected = null;
   bodegaSelect.value.options.push(...bodegas);
-  step.value++;
+  if (bodegaSelect.value.options.length == 1) {
```

Comment:
This login could be change with a simple if with return statement, something like this:

```
if(bodegaSelect.value.options.length == 1) {
  // change the value of the states
  step.value = 3;
  return;
}

// add the logic if the user has more than 1 warehouse
```